manifests/init.pp

```diff
@@ -40,14 +40,17 @@
   # middleware connector tweaking
   $rabbitmq_vhost = '/mcollective',
 
+  # common
+  $common_package = 'mcollective-common',
+
   # server-specific
-  $server_config_file = undef, # default dependent on $confdir
-  $server_logfile     = '/var/log/mcollective.log',
-  $server_loglevel    = 'info',
-  $server_daemonize   = 1,
-  $service_name       = 'mcollective',
-  $server_package     = 'mcollective',
-  $ruby_stomp_package = 'ruby-stomp',
+  $server_config_file    = undef, # default dependent on $confdir
```

Comment:
Right.
If you have look at the previous commits, I added this line : 
`$server_package_common = 'mcollective-common',`
(and so I re-indented the other lines)

I removed this line in the next commit and I forgot to indent properly...
See https://github.com/RewardGateway/puppetlabs-mcollective/commit/50a311572328965101d95b17c7b128a73b91660a

Do you want I push a new commit with the correct indentation ?